components/delete.js

```diff
@@ -71,6 +73,7 @@ function DeleteConfirm ({ onConfirm }) {
           variant='danger' onClick={async () => {
             try {
               await onConfirm()
+              toaster.success('deleted item')
```

Comment:
@ekzyis AFAICT this is only used in the dropdown for an item, so I didn't make the message generic to support multiple item types.

Reply:
I can't check right now but I think this also shows up when you remove an authentication method. (accidentally used "account" instead of "auth method" in my previous comment)

Reply:
It has its own implementation. I updated the PR to add toast to the auth method operations, too.